src/RepoAuditor/CommandLineProcessor.py

```diff
@@ -169,13 +169,13 @@ def Create(
 
         for key, value in get_dynamic_args_func(dynamic_arg_definitions).items():
             parts = key.split(argument_separator)
-            assert len(parts) >= 2
+            assert len(parts) >= 2  # noqa: PLR2004

```

Comment:
I think we should be able to get rid / tighten this `assert`, but I need to dive into the subsequent lines to understand the reason behind this assert.